packages/orchestration/src/orchestration-types.ts

```diff
@@ -361,9 +361,35 @@ export type DpiMaskingConfig = Omit<
 };
 
 /**
- * Filter configuration for Azure content safety Filter.
+ * Input parameters for Azure content safety input filter.
  */
-export interface AzureContentFilter {
+export interface AzureContentSafetyFilterInputParameters {
+  /**
+   * The filter category for hate content.
+   */
+  hate?: AzureFilterThreshold;
+  /**
+   * The filter category for self-harm content.
+   */
+  self_harm?: AzureFilterThreshold;
+  /**
+   * The filter category for sexual content.
+   */
+  sexual?: AzureFilterThreshold;
+  /**
+   * The filter category for violence content.
+   */
+  violence?: AzureFilterThreshold;
```

Comment:
[pp] Consider having a base interface for common properties shared between the two.